src/QualificationRunner.Core/Constants.cs

```diff
@@ -14,9 +14,11 @@ public static class Constants
       public const string CONTENT_FOLDER = "Content";
       public const string OBSERVED_DATA_FOLDER = "ObservedData";
       public const string INPUTS_FOLDER = "Inputs";
+      public const string INTRO_FOLDER = "Intro";

```

Comment:
This is where all intro files will be exported. That way, the reporting engine does not have to do anything except movign all those files in the output folder